README.md

```diff
@@ -171,13 +148,7 @@ For more information about how to use the `ofn demo` command, refer to [ofn demo
 
 ### Uninstall OpenFunction
 
-Run the following command to uninstall OpenFunction and its dependencies.
-
-```shell
-ofn uninstall --all
-```
-
-For more information about how to use the `ofn uninstall` command, refer to [ofn uninstall document](https://github.com/OpenFunction/cli/blob/main/docs/uninstall.md).
+See our [Uninstallation](https://openfunction.dev/docs/getting-started/installation/#uninstall-openfunction) guide over in our docs.
```

Comment:
To uninstall OpenFunction, please refer to [Uninstallation](https://openfunction.dev/docs/getting-started/installation/#uninstall-openfunction)